doc/api/async_hooks.md

```diff
@@ -272,24 +273,25 @@ elaborate to make calling context easier to see.
 
 ```js
 const async_hooks = require('async_hooks');
+const fs = require('fs');
 
 let indent = 0;
 async_hooks.createHook({
   init(asyncId, type, triggerId) {
     const cId = async_hooks.currentId();
-    fs.writeSync(1, ' '.repeat(indent) + `${type}(${asyncId}): ` +
+    fs.writeSync(1, `${' '.repeat(indent)}${type}(${asyncId}): ` +
```

Comment:
FWIW You know I like it when you assign expressions to `const`s, make it "less operations per line"
But if not I'm +1 for homogeneous templates vs. templates + contacts